core/src/test/scala/pickling/neg/java-field-fail.scala

```diff
@@ -0,0 +1,20 @@
+package scala.pickling.javafieldfail
+
+import scala.pickling._
+import NegativeCompilation._
+import org.scalatest.FunSuite
+
+class JavaFieldFailTest extends FunSuite {
+  test("main") {
+    expectError("Cannot generate") {
```

Comment:
this is the `implicitNotFound` error right - it might be better to try to compile an explicit `Pickler.generate[java.lang.Byte]` and check for the specific PicklingException you added ?